code/nodes/saveGraph/utils.q

```diff
@@ -0,0 +1,138 @@
+\d .automl
+
+// Utility functions for saveGraph
+
+// Python functionality
+plt:.p.import`matplotlib.pyplot;
+
+// @kind function
+// @category saveGraphUtility
+// @fileoverview Create regression target distribution plot and save down locally
+// @param target    {float[]} All target values
+// @param savePath  {int} Path to where plots are to be saved
+// @return {null} Target distribution plot saved to appropriate location
+saveGraph.i.regTargetPlot:{[target;savePath]
+  plt[`:hist][target;`bins pykw 10;`ec pykw"black"];
+  plt[`:title]["Target Distribution";`fontsize pykw 12];
+  plt[`:xlabel]["Target"];
+  plt[`:ylabel]["Count"];
+  filePath:savePath,"Target_Distribution.png";
+  plt[`:savefig][filePath;`bbox_inches pykw"tight"];
+  plt[`:close][];
+  }
+
+// @kind function
+// @category saveGraphUtility
+// @fileoverview Create binary target distribution plot and save down locally
+// @param target    {float[]} All target values
+// @param savePath  {int} Path to where plots are to be saved
+// @return {null} Target distribution plot saved to appropriate location
+saveGraph.i.classTargetPlot:{[target;savePath]
```

Comment:
May be worth reintroducing the target names if they were originally symbols here, it'll give a more explicit context to the distributions if they're non binary or physical in nature.